src/webextension/widgets/scrolling-list.css

```diff
@@ -26,4 +26,5 @@
 .scrolling-list:focus > .styled-item[data-selected] {
   color: #ffffff;
   background-color: #0060df;
+  box-shadow: inset 0 1px 0 #0a84ff, inset 0 -1px 0 #0a84ff;
```

Comment:
I don't think we need this change (at least not yet). For the time being, my intention is that a `<ScrollingList/>` with styled items should look like the search engine list in `about:preferences#search`. (This implementation makes it easier to see the focus state, though.)